scripts/create_mortgage_and_request_validation.sh

```diff
@@ -0,0 +1,269 @@
+###############################################################################
```

Comment:
I assume this script is for demo purposes? Can we consider doing what AC has done (https://github.com/FigureTechnologies/asset-classification-verifier) by moving this stuff into gradle tasks?